sanity-io/structured-content-2022
# Adjust grid, breakpoints

# Adjust grid, breakpoints

## Intent

Adjust the "grid" size and the "responsive" breakpoints according to [designer input in Slack](https://wja.slack.com/archives/C02E2ERKR7E/p1645183048325309)

## Description

This is just adjusting two constants used by the current CSS for overall layout and media query "breakpoints".
- The styling that's based on the "desktop" Figma sketches is now used for more viewport sizes: 1025px and up (i.e. some viewport sizes that are narrower than the Figma sketch shows) instead of 1440px and up
- In the "desktop" layout, the content (that's forming a 12-column "grid" in the design) is now allowed to grow to max 1440px width instead of 1360 which it was previously (so it is allowed to grow a bit wider than it is in the Figma sketch before we start capping it)

## Testing this PR

1. Open https://structured-content-2022-web-git-adjust-grid-breakpoints.sanity.build/home
2. Make the window very wide and verify via the browser's dev tools that the "STRUCTURED CONTENT 2022" image is 1440px wide
3. Shrink the window to about 1100px (again with the help of dev tools to measure) and verify that the NavBlock (yellow links and boxes a bit further down on the page) still shows 3 empty blocks next to "Program", not 2
